app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,14 @@
+  <% if notice.present? %>
+    <p><%= notice %></p>
+  <% end %>
+  <h1>Recipes</h1>
+  <div>
+    <% if @recipes.any? %>
+      <% @recipes.each do |recipe| %>
+        <%= render recipe %>
+      <% end %>
+    <% else %>
+      <p>No recipes in database.</p>
+    <% end %>
+  </div>
+  <%= link_to "Add New recipe", new_recipe_path %>
```

Comment:
- [ ] It would be nice if we keep to the given wireframe
your design :point_down: 
![Screenshot from 2023-12-02 12-37-42](https://github.com/Kaiserabbas/food_recipe/assets/66097371/c86aba8d-2978-44a5-bd21-f7684d903ac8)
given wireframe :point_down: 
![Screenshot from 2023-12-02 12-38-58](https://github.com/Kaiserabbas/food_recipe/assets/66097371/b82826c6-42e8-46f2-8dd7-33ad476fadb3)